tests/Jobs/GenerateImageAnnotationPatchTest.php

```diff
@@ -234,6 +295,10 @@ public function testHandleError()
         $annotation->shape_id = Shape::pointId();
         $job = new GenerateImageAnnotationPatch($annotation);
         $job->handle();
+
+        $prefix = fragment_uuid_path($annotation->image->uuid);
+        $content = Storage::disk('test')->get("{$prefix}/{$annotation->id}.svg");
+        $this->assertEquals(null, $content);
```

Comment:
Just so you know, there is also `$disk->assertMissing('path')` or `$this->assertNull($content)`. But you don't have to change this here as it works just as well.